go-controller/hack/lint.sh

```
@@ -1,14 +1,18 @@
 if [ "$#" -ne 1 ]; then
+  if [ "$#" -eq 2 ] && [ "$2" == "fix" ]; then
+    extra_flags="--fix"
+  else
     echo "Expected command line argument - container runtime (docker/podman) got $# arguments: $@"
     exit 1
+  fi
 fi
 
 $1 run --security-opt label=disable --rm \
   -v  ${HOME}/.cache/golangci-lint:/cache -e GOLANGCI_LINT_CACHE=/cache \
   -v $(pwd):/app -w /app -e GO111MODULE=on docker.io/golangci/golangci-lint:${VERSION} \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)